stripe/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -539,73 +551,46 @@ class CardInputWidget @JvmOverloads constructor(
         cardNumberEditText.requestFocus()
     }
 
+    // reveal the full card number field
     private fun scrollLeft() {
         if (cardNumberIsViewed || !initFlag) {
             return
         }
 
-        val dateStartPosition = placementParameters.peekCardWidth +
-            placementParameters.cardDateSeparation
-        val cvcStartPosition = (dateStartPosition +
-            placementParameters.dateWidth + placementParameters.dateCvcSeparation)
+        val dateStartPosition = placementParameters.dateDestination
```

Comment:
is this intentional? date starts at its destination and doesn't animate?

Reply:
This is a no-op because `placementParameters.dateDestination` is defined as

```
        internal val dateDestination: Int
            @JvmSynthetic
            get() {
                return peekCardWidth + cardDateSeparation
            }
```

https://github.com/stripe/stripe-android/pull/1920/files#diff-e68e41550603be48c74435f62f8bee4fR768

Reply:
I ended up renaming the methods in `PlacementParameters` to be explain their usage